tests/mqtt-frames-xpdu/test.yaml

```diff
@@ -0,0 +1,89 @@
+pcap: ../mqtt5-unsub-userpass/input.pcap
+
+requires:
+  min-version: 8
+
+args:
+ - -k none
+ - --set outputs.1.eve-log.types.1.frame.enabled=yes
```

Comment:
@jasonish is this robust enough ?

Alternative is to use own suricata.yaml, right ?
Or not have frames output, just alerts...

Reply:
Yeah, but should we ever overhaul the default config it will break.. But so will many many tests.